.config/notice-file/config.yaml

```diff
@@ -1,12 +1,10 @@
 ---
-
 title: "Mattermost Calls"
 copyright: "©2015-present Mattermost,Inc. All Rights Reserved. See LICENSE for license information."
 description: "This document includes a list of open source components used in Mattermost Calls plugin, including those that have been modified."
 search:
   - "go.mod"
   - "build/go.mod"
+  - "webapp/package.json"
```

Comment:
Technically speaking we also have `standalone/package.json` which mostly overlaps with `webapp/package.json` but it's a different thing. We also have `server/public/go.mod` and `lt/go.mod`.

Reply:
We mainly need the ones that are actually used on production . Which files depict those dependencies ?

Reply:
All of those, except `lt/go.mod` would be part of our builds.

Reply:
From the ones there already, `build/go.mod` wouldn't really be part of our releases. It's just a dev dependency.